core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -5932,20 +5932,40 @@ components:
       type: object
       required:
         - network
-        - payload_hash
+        - payload_hashes
       properties:
         network:
           $ref: "#/components/schemas/NetworkIdentifier"
-        payload_hash:
-          $ref: "#/components/schemas/NotarizedTransactionHash"
+        payload_hashes:
+          type: array
+          items:
+            $ref: "#/components/schemas/NotarizedTransactionHash"
     MempoolTransactionResponse:
       type: object
       required:
-        - payload_hex
+        - count
+        - payloads
       properties:
-        payload_hex:
-          type: string
-          description: The hex-encoded full notarized transaction payload.
+        count:
+          type: integer
+          format: int32
+          description: An integer giving the total count of payload hashes checked in the returned response.
+        payloads:
```

Comment:
We better indicate somewhere in apidocs that `payloads` may be cut short because of reasons.
(it is unusual for a batch "list of IDs" endpoint to silently ignore _some_ of the explicitly given IDs)